Tasks/Gradle/Gradle.ps1

```diff
@@ -100,7 +100,7 @@ $subprojects = Invoke-BatchScript -Path $wrapperScript -Arguments 'properties' -
 Write-Verbose "subprojects: $subprojects"
 $singlemodule = [string]::IsNullOrEmpty($subprojects) -or $subprojects -eq '[]'
 
-if($codeCoverageTool.equals("Jacoco"))
+if($codeCoverageTool.equals("JaCoCo"))
```

Comment:
lets do a ignore care for string comparisons so that we dont run into this anywhere else.
use -eq or add second param=1 in the equals function to get this